index.html

```diff
@@ -297,7 +293,7 @@
   </p>
 </li>
 <li><a href="http://www.w3.org/TR/html5/webappapis.html#queue-a-task">Queue a task</a> on the queue associated with the idle-task task
-source, which performs the <a>invoke idle callback timeout algorithm</a>, passing <var>handle</var> and <var>document</var> as arguments.
```

Comment:
I think we need it in order to get the right "list of idle request callbacks" or the "list of runnable idle callbacks" in which to lookup the callback associated with "handle". See comment below.